### PR TITLE
[MODEVENTC-17] RMB version update to 29.0.1

### PR DESCRIPTION
**Pull request by kostyantyn-kh**

### CODE REVIEW

**Comment by kostyantyn-kh** [comment]
Most likely i'll create separate story for test coverage improvement

**Comment by kostyantyn-kh** [comment]
@adamdickmeiss  mapExceptionToResponse test was added

File: src/main/java/org/folio/rest/impl/util/EventConfigHelper.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.folio.cql2pgjson.exception.CQL2PgJSONException;
+import org.folio.rest.persist.cql.CQLQueryValidationException;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+public class EventConfigHelper {
```
Comment (julianladisch) [inline src/main/java/org/folio/rest/impl/util/EventConfigHelper.java]: Can you add a private constructor?
https://sonarcloud.io/organizations/folio-org/rules?open=squid%3AS1118&rule_key=squid%3AS1118

File: src/main/java/org/folio/rest/impl/EventConfigAPIs.java [hunk 2]
```diff
@@ -3,6 +3,7 @@
 import io.vertx.core.*;
 import io.vertx.core.logging.Logger;
 import io.vertx.core.logging.LoggerFactory;
```
Comment (julianladisch) [inline src/main/java/org/folio/rest/impl/EventConfigAPIs.java]: Can you remove the unused imports?

File: src/test/java/org/folio/impl/util/EventConfigHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    response = EventConfigHelper.mapException(new CQLQueryValidationException(null));
+    assertEquals(response.getStatus(), HttpStatus.SC_BAD_REQUEST);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+
+    response = EventConfigHelper.mapException(new NullPointerException());
+    assertEquals(response.getStatus(), HttpStatus.SC_INTERNAL_SERVER_ERROR);
+    assertEquals(response.getMediaType().toString(), MediaType.TEXT_PLAIN);
+  }
+
+}
```
Comment (julianladisch) [inline src/test/java/org/folio/impl/util/EventConfigHelperTest.java]: There are two possibilities to assert for equality:
* `assertEquals(expected, actual)`
* `assertThat(actual, is(expected))` from org.hamcrest.MatcherAssert.assertThat

They differ in the order, the first one has `expected, actual`, the second `actual, expected`.

Please swap the parameters if you want to use `assertEquals`, or switch to `assertThat`.

File: src/main/java/org/folio/rest/impl/util/EventConfigHelper.java (hunk 1 above)
Comment (kostyantyn-kh) [reply src/main/java/org/folio/rest/impl/util/EventConfigHelper.java]: Added

File: src/main/java/org/folio/rest/impl/EventConfigAPIs.java (hunk 2 above)
Comment (kostyantyn-kh) [reply src/main/java/org/folio/rest/impl/EventConfigAPIs.java]: Unused imports removed

File: src/test/java/org/folio/impl/util/EventConfigHelperTest.java (hunk 3 above)
Comment (kostyantyn-kh) [reply src/test/java/org/folio/impl/util/EventConfigHelperTest.java]: Fixed, thank you

**Pull request merged**